packages/core/test/combinator/multicast-test.js

```diff
@@ -0,0 +1,37 @@
+import { describe, it } from 'mocha'
+import { assert, eq } from '@briancavalier/assert'
+import { spy } from 'sinon'
+
+import { now } from '../../src/source/now'
+import { map } from '../../src/combinator/transform'
+import { multicast } from '../../src/combinator/multicast'
+import { observe } from '../helper/observe'
+
+describe('multicast', () => {
+  it('should be identity for already-multicasted stream', () => {
+    const s = multicast(now(1))
+    eq(s, multicast(s))
+  })
+
+  it('should call mapper function once when there are > 1 observer', () => {
+    const mapperSpy = spy()
+    const observer1Spy = spy()
+    const observer2Spy = spy()
+    const observer3Spy = spy()
+
+    const s = now({value: 'sentinel'})
```

Comment:
When I saw `{ value: 'sentinel' }`, I went looking for an assertion about the the value.  Since the event value is never used, it could something like `1` or `{}`.